### ISSUE TITLE
Use a proper daemon

### ISSUE THREAD

**Issue by JadedCtrl** [JadedCtrl/Pogger#11]
Pogger should follow the daemon/deskbar replicant/settings model, instead of having the MainWindow be open all the time. A few changes'll have to be made.

- Window
When settings are changed, a message will have to be sent to the daemon saying as much.
A "Run Pogger on system startup" option should be added.

- Deskbar Replicant
The mail service has its deskbar replicant manage notifications etc (it consistently runs a query for new mail), and Pogger's deskbar replicant should follow suit. Right now Notifier requires messages from the update loop to send notifications― so if Notifier becomes a child of the DeskbarView, then it'll have to move to using queries, like mail's.

- Daemon
Basically just has a FeedController do all the work.
